app/src/main/index.js

```diff
@@ -399,11 +399,13 @@ const eventHandlers = {
   },
 
   "error-collection": (event, optin) => {
-    Raven.uninstall()
-      .config(optin ? config.sentry_dsn : ``, {
-        captureUnhandledRejections: false
+    if (optin) {
+      Sentry.init({
+        dsn: config.sentry_dsn,
+        release: `voyager@0.8001.1`
```

Comment:
Yeah it's WIP but I was curious :) 
If you want to forget aligning this with package.json we can do something like

```javascript
import {version} from './package.json';
```

Reply:
good idea. will fix.